src/test/java/com/puppycrawl/tools/checkstyle/checks/naming/LocalFinalVariableNameCheckTest.java

```diff
@@ -56,9 +58,11 @@ public void testDefault()
         final String pattern = "^[a-z][a-zA-Z0-9]*$";
 
         final String[] expected = {
-            "123:19: " + getCheckMessage(MSG_INVALID_PATTERN, "CDE", pattern),
+            "17:19: " + getCheckMessage(MSG_INVALID_PATTERN, "CDE", pattern),
```

Comment:
Again, we are not expecting changes to this test.
Changes for your issue should be in an input file that no other tests uses to avoid this conflict.
Since this feature will increase violations, we are expecting violations for old tests to stay the same or increase. We don't expect any to disappear which is what is happening to `123`.